sources/install/package_wordlists.sh

```diff
@@ -106,6 +106,45 @@ function install_genusernames() {
     add-to-list "genusernames,https://gitlab.com/-/snippets/2480505/raw/main/bash,GenUsername is a Python tool for generating a list of usernames based on a name or email address."
 }
 
+function install_gensymlinks() {
+    # Create Symlinks of all wordlist on Exegol to /opt/list
+    colorecho "Generate Symlinks"
+    mkdir -p /opt/lists
```

Comment:
let's move this folder creation in the `filesystem()` function in `package_base.sh` and remove that `install_gensymlinks()` function